libraries/azure_power_bi_capacity_workload.rb

```diff
@@ -0,0 +1,31 @@
+require 'azure_generic_resource'
+
+class AzurePowerBICapacityWorkload < AzureGenericResource
+  name 'azure_power_bi_capacity_workload'
+  desc 'Retrieves and verifies the settings of an Azure Power BI Capacity Workload.'
+  example <<-EXAMPLE
+    describe azure_power_bi_capacity_workload(capacity_id: '0f084df7-c13d-451b-af5f-ed0c466403b2', name: 'cfafbeb1-8037-4d0c-896e-a46fb27ff229') do
```

Comment:
```suggestion
    describe azure_power_bi_capacity_workload(capacity_id: 'CAPACITY_ID', name: 'WORKLOAD_NAME') do
```

Reply:
no here it has be original values in example, only in docs we specify that notation!

Reply:
Thanks for the information.